net/uradvd/files/etc/init.d/uradvd

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh /etc/rc.common
+
+START=50
+
+USE_PROCD=1
+
+start_instance() {
+	local cfg="$1" enabled device ifname default_lifetime args
+
+	config_get enabled $cfg 'enabled'
+	config_get device $cfg 'device'
+	config_get ifname $cfg 'ifname'
+	config_get default_lifetime $cfg 'default_lifetime'
+
+	if [ "$enabled" != "1" ]; then
+		echo "disabled in config"
```

Comment:
I'd avoid printing anything when an instance is disabled.

Reply:
I kinda disagree here. When the uradvd is enabled via /etc/rc.d/, but doesn't start, then there should be a log message.
Anyway, I removed it, as I do not care that much about his detail.